app/uk/gov/hmrc/plasticpackagingtax/registration/views/enrolment/reference_number_already_used_failure_page.scala.html

```diff
@@ -0,0 +1,47 @@
+@*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.plasticpackagingtax.registration.controllers.enrolment.{routes => pptRoutes}
+@import uk.gov.hmrc.plasticpackagingtax.registration.views.html.main_template
+@import uk.gov.hmrc.plasticpackagingtax.registration.views.model.Title
+@import uk.gov.hmrc.plasticpackagingtax.registration.config.AppConfig
+
+@this(
+        formHelper: FormWithCSRF,
+        govukLayout: main_template,
+        paragraphBody: paragraphBody,
+        pageHeading: pageHeading,
+        govukButton: GovukButton,
+        appConfig: AppConfig
+)
+
+@()(implicit request: Request[_], messages: Messages)
+
+@govukLayout(title = Title("enrolment.referenceNumber.already.used.title")) {
+
+    @formHelper(action =Call("GET", appConfig.businessAccountUrl), 'autoComplete -> "off") {
+
+        @pageHeading(messages("enrolment.referenceNumber.already.used.title"))
+
+        @paragraphBody(message = messages("enrolment.referenceNumber.already.used.detail1"))
+
+        @paragraphBody(message = messages("enrolment.referenceNumber.already.used.detail2"))
+
+        @govukButton(Button(
```

Comment:
Interesting way of doing this.   I'd have done something like rendering a link to `appConfig.businessAccountUrl` to look like a button (there is a govuk css class to do this)